interfaces/cython/SConscript

```diff
@@ -172,7 +170,7 @@ if localenv['python_package'] == 'full':
     py2env['py_extension'] = ext[0].name
     py2env.SubstFile('setup2.py', 'setup.py.in')
     build_cmd = ('cd interfaces/cython &&'
-                 ' $python_cmd_esc setup2.py build --build-lib=../../build/python2')
+                 ' $python2_cmd setup2.py build --build-lib=../../build/python2')
```

Comment:
I think we will continue to need to use a quoted/escaped Python command for the same reason it was originally introduced (see 4f3dd0624)

Reply:
But the Python 3 interface does not use the quoted command. Should that be changed to use a quoted command?

Reply:
Yes, presumably. I guess the quoting can be done more locally than in the current version.

Reply:
Shouldn't we then use the escaped command everywhere? Like, all the places we check for NumPy, in the tests, etc?